src/manifest/owned/mod.rs

```diff
@@ -0,0 +1,412 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+//! Owned manifest containers.
+//!
+//! TODO(mcyoung): Document this module. This should be done in a followup
```

Comment:
`TODO(issue#)`.

Reply:
This was actually done in a followup commit. =)